packages/suite/src/utils/suite/guide.ts

```diff
@@ -0,0 +1,38 @@
+import { GuideCategoryType, GuideItemType } from '@suite-types/guide';
+
+export const getItemTitle = (item: GuideItemType, language: string): string =>
+    item.title[language] || item.title.en;
+
+export const findParentCategories = (
+    item: GuideItemType,
+    root: GuideCategoryType,
+): GuideCategoryType[] | null => {
+    const itemParentId = item.id.split('/').slice(0, -1).join('/');
+
+    const findParentsRecursive = (
+        index: number,
+        item: GuideCategoryType,
+        searchedId: string,
+    ): GuideCategoryType[] | null => {
+        const currentItemId = `${searchedId
+            .split('/')
+            .slice(0, index + 1)
+            .join('/')}`;
+
+        const itemToLook = (item.children as any[]).find(i => i.id === currentItemId) as
```

Comment:
if you could switch the type of children as I suggested here: https://github.com/trezor/trezor-suite/pull/3962/files#r661411950 than it could work here without those ugly `as`

Reply:
Resolved.